runtests_parser.py

```diff
@@ -0,0 +1,121 @@
+import json
+
+class Test:
+    def __init__(self, id, desc):   
+        self.id = id 
+        self.desc = desc.strip()
+        self.more_vw = False
+        self.vw_command = "" 
+        self.is_bash_command = False
+        self.files = []
+
+    def add_more_comments(self, line):
+        self.desc = self.desc + ". " + line.strip()[1:].strip()
+    
+    def add_vw_command(self, line):
+        self.vw_command = self.vw_command + " " + line
+        if line[-1] == "\\":
+            self.vw_command = self.vw_command[:-1]
+            self.more_vw = True
+
+        if "&&" in self.vw_command:
+            self.is_bash_command = True
+    
+    def force_vw_append(self, line):
+        if self.more_vw:
+            self.more_vw = False
+            self.add_vw_command(line)
+            return True
+        else:
+            return False
+        
+    def add_file(self, line):
+        self.files.append(line)
+
+    def add_bash_command(self, line):
+        if self.vw_command:
+            raise Exception("construction faulty merging" + self.vw_command + " and " + line)
+        self.is_bash_command = True
+        self.add_vw_command(line)
+
+    def clean(self):
+        if not self.is_bash_command:
+            self.vw_command = " ".join(self.vw_command.split()[1:])
+        delattr(self, 'more_vw')
+
+class Parser:
+    def __init__(self):   
+        self.temp_test = None
+        self.results = []
+
+    @staticmethod
+    def is_perl_comment(line):
+        return line[0] == "#"
+
+    @staticmethod
+    def is_vw_command(line):
+        return line.split()[0] == "{VW}"
```

Comment:
Does this handle multiline commands with continuations? "\"

Reply:
Yes, but its more of a state machine rather than trying to foresee. It's handled via more_vw flag, basically checking if line ends with `\` and then parsing as such on line 98